examples/diffeq.jl

```diff
@@ -83,7 +83,8 @@ end
                         prob;
                         p = [σ, ρ, β],
                         tspan = (ts[1], ts[end])
-                    )
+                    ),
+                    Tsit5(),
```

Comment:
Is this change intentional?

Reply:
Yeah, had to make a couple fixes when I regenerated the reference image set, so I figured I'd chuck those in here as well.